`build` fails if MLEM is installed with `pip install -e .`

(btw, these warnings are too noisy I think)
Reproducing:

```
$ git clone https://github.com/iterative/mlem
$ cd mlem
$ pip install -e .

$ mlem --tb build docker -m ../mlem/emoji/lyrics2emoji --image.name lyrics2emoji --image.tag v1
⏳️ Loading model from ../mlem/emoji/lyrics2emoji.mlem
2022-12-28 16:37:30,328 [WARNING] mlem.utils.module: Fail to determine package name for module 'spacy', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:30,440 [WARNING] mlem.utils.module: Fail to determine package name for module 'transformers', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:30,744 [WARNING] mlem.utils.module: Fail to determine package name for module 'torch', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:31,649 [WARNING] mlem.utils.module: Fail to determine package name for module 'click<8.2', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:31,853 [WARNING] mlem.utils.module: Fail to determine package name for module 'aiohttp<4', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:31,960 [WARNING] mlem.utils.module: Fail to determine package name for module 'aiohttp_swagger<2', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:32,063 [WARNING] mlem.utils.module: Fail to determine package name for module 'Jinja2', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:32,300 [WARNING] mlem.utils.module: Fail to determine package name for module 'pyparsing<4', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:32,404 [WARNING] mlem.utils.module: Fail to determine package name for module 'cached-property', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:32,616 [WARNING] mlem.utils.module: Fail to determine package name for module 'gitpython', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:32,721 [WARNING] mlem.utils.module: Fail to determine package name for module 'python-gitlab', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:32,927 [WARNING] mlem.utils.module: Fail to determine package name for module 'iterative-telemetry', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:33,031 [WARNING] mlem.utils.module: Fail to determine package name for module 'python-multipart', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:33,134 [WARNING] mlem.utils.module: Fail to determine package name for module 'importlib-metadata', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
🛠 Building MLEM wheel file...
💼 Adding model files...
2022-12-28 16:37:49,316 [WARNING] mlem.utils.module: Fail to determine package name for module 'spacy', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:49,479 [WARNING] mlem.utils.module: Fail to determine package name for module 'transformers', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
2022-12-28 16:37:49,811 [WARNING] mlem.utils.module: Fail to determine package name for module 'torch', using module name instead. If that's incorrect, fix metadata manually, either in `.mlem` file or in your MLEM Python object.
🛠 Generating dockerfile...
💼 Adding sources...
💼 Generating requirements file...
🛠 Building docker image lyrics2emoji:v1...
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils: Step 1/12 : FROM python:3.9.5-slim
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils:
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils: ---> c71955050276
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils: Step 2/12 : WORKDIR /app
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils:
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils: ---> Using cache
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils: ---> cc67e297a6e5
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils: Step 3/12 : COPY requirements.txt .
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils:
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils: ---> Using cache
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils: ---> 658275b32126
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils: Step 4/12 : RUN pip install -r requirements.txt && pip cache purge
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils:
2022-12-28 16:38:01,631 [ERROR] mlem.contrib.docker.utils: ---> Using cache
2022-12-28 16:38:01,632 [ERROR] mlem.contrib.docker.utils: ---> 47818fcd0d20
2022-12-28 16:38:01,632 [ERROR] mlem.contrib.docker.utils: Step 5/12 : COPY mlem_requirements.txt .
2022-12-28 16:38:01,632 [ERROR] mlem.contrib.docker.utils:
2022-12-28 16:38:01,632 [ERROR] mlem.contrib.docker.utils: ---> Using cache
2022-12-28 16:38:01,632 [ERROR] mlem.contrib.docker.utils: ---> f227852042d5
2022-12-28 16:38:01,632 [ERROR] mlem.contrib.docker.utils: Step 6/12 : RUN pip install -r mlem_requirements.txt && pip cache purge
2022-12-28 16:38:01,641 [ERROR] mlem.contrib.docker.utils:
2022-12-28 16:38:01,641 [ERROR] mlem.contrib.docker.utils: ---> Using cache
2022-12-28 16:38:01,641 [ERROR] mlem.contrib.docker.utils: ---> c11db71e6f7e
2022-12-28 16:38:01,641 [ERROR] mlem.contrib.docker.utils: Step 7/12 : COPY mlem-0.4.1.dev2+gd4e0ddb-py3-none-any.whl .
2022-12-28 16:38:01,641 [ERROR] mlem.contrib.docker.utils:
2022-12-28 16:38:01,641 [ERROR] mlem.contrib.docker.utils: ---> a20710eec288
2022-12-28 16:38:01,641 [ERROR] mlem.contrib.docker.utils: Step 8/12 : RUN pip install mlem-0.4.1.dev2+gd4e0ddb-py3-none-any.whl && pip cache purge
2022-12-28 16:38:01,641 [ERROR] mlem.contrib.docker.utils:
2022-12-28 16:38:01,641 [ERROR] mlem.contrib.docker.utils: ---> Running in c61ce4ef859d
2022-12-28 16:38:01,641 [ERROR] mlem.contrib.docker.utils: Processing ./mlem-0.4.1.dev2+gd4e0ddb-py3-none-any.whl
2022-12-28 16:38:01,641 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: pyparsing<4 in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (3.0.9)
2022-12-28 16:38:01,642 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: flatdict in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (4.0.1)
2022-12-28 16:38:01,642 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: typer in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (0.7.0)
2022-12-28 16:38:01,642 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: aiohttp<4 in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (3.8.3)
2022-12-28 16:38:01,642 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: cached-property in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (1.5.2)
2022-12-28 16:38:01,642 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: fsspec>=2021.7.0 in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (2021.7.0)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: importlib-metadata in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (5.2.0)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: pydantic<2,>=1.9.0 in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (1.9.0)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: python-multipart in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (0.0.5)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: rich in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (12.6.0)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: requests in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (2.28.1)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: isort>=5.10 in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (5.10.0)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: dill in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (0.3.6)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: gitpython in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (3.1.29)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: python-gitlab in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (3.12.0)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: cloudpickle in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (2.2.0)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: Jinja2>=3 in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (3.0.0)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: click<8.2 in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (8.1.3)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: entrypoints in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (0.4)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: aiohttp-swagger<2 in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (1.0.16)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: iterative-telemetry>=0.0.6 in /usr/local/lib/python3.9/site-packages (from mlem==0.4.1.dev2+gd4e0ddb) (0.0.6)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.9/site-packages (from aiohttp<4->mlem==0.4.1.dev2+gd4e0ddb) (1.3.1)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.9/site-packages (from aiohttp<4->mlem==0.4.1.dev2+gd4e0ddb) (6.0.4)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /usr/local/lib/python3.9/site-packages (from aiohttp<4->mlem==0.4.1.dev2+gd4e0ddb) (4.0.2)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.9/site-packages (from aiohttp<4->mlem==0.4.1.dev2+gd4e0ddb) (1.3.3)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: charset-normalizer<3.0,>=2.0 in /usr/local/lib/python3.9/site-packages (from aiohttp<4->mlem==0.4.1.dev2+gd4e0ddb) (2.1.1)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.9/site-packages (from aiohttp<4->mlem==0.4.1.dev2+gd4e0ddb) (22.2.0)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.9/site-packages (from aiohttp<4->mlem==0.4.1.dev2+gd4e0ddb) (1.8.2)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: pyYAML>=5.1 in /usr/local/lib/python3.9/site-packages (from aiohttp-swagger<2->mlem==0.4.1.dev2+gd4e0ddb) (6.0)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: appdirs in /usr/local/lib/python3.9/site-packages (from iterative-telemetry>=0.0.6->mlem==0.4.1.dev2+gd4e0ddb) (1.4.4)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: distro in /usr/local/lib/python3.9/site-packages (from iterative-telemetry>=0.0.6->mlem==0.4.1.dev2+gd4e0ddb) (1.8.0)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: filelock in /usr/local/lib/python3.9/site-packages (from iterative-telemetry>=0.0.6->mlem==0.4.1.dev2+gd4e0ddb) (3.8.2)
2022-12-28 16:38:01,643 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: MarkupSafe>=2.0.0rc2 in /usr/local/lib/python3.9/site-packages (from Jinja2>=3->mlem==0.4.1.dev2+gd4e0ddb) (2.1.1)
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: typing-extensions>=3.7.4.3 in /usr/local/lib/python3.9/site-packages (from pydantic<2,>=1.9.0->mlem==0.4.1.dev2+gd4e0ddb) (4.4.0)
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: idna>=2.0 in /usr/local/lib/python3.9/site-packages (from yarl<2.0,>=1.0->aiohttp<4->mlem==0.4.1.dev2+gd4e0ddb) (3.4)
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: gitdb<5,>=4.0.1 in /usr/local/lib/python3.9/site-packages (from gitpython->mlem==0.4.1.dev2+gd4e0ddb) (4.0.10)
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: smmap<6,>=3.0.1 in /usr/local/lib/python3.9/site-packages (from gitdb<5,>=4.0.1->gitpython->mlem==0.4.1.dev2+gd4e0ddb) (5.0.0)
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.9/site-packages (from importlib-metadata->mlem==0.4.1.dev2+gd4e0ddb) (3.11.0)
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: requests-toolbelt>=0.9.1 in /usr/local/lib/python3.9/site-packages (from python-gitlab->mlem==0.4.1.dev2+gd4e0ddb) (0.10.1)
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/local/lib/python3.9/site-packages (from requests->mlem==0.4.1.dev2+gd4e0ddb) (1.26.13)
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.9/site-packages (from requests->mlem==0.4.1.dev2+gd4e0ddb) (2022.12.7)
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: six>=1.4.0 in /usr/local/lib/python3.9/site-packages (from python-multipart->mlem==0.4.1.dev2+gd4e0ddb) (1.16.0)
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in /usr/local/lib/python3.9/site-packages (from rich->mlem==0.4.1.dev2+gd4e0ddb) (0.9.1)
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: Requirement already satisfied: pygments<3.0.0,>=2.6.0 in /usr/local/lib/python3.9/site-packages (from rich->mlem==0.4.1.dev2+gd4e0ddb) (2.13.0)
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: Installing collected packages: mlem
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: Successfully installed mlem-0.4.1.dev2+gd4e0ddb
2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv

2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: WARNING: You are using pip version 21.1.3; however, version 22.3.1 is available.
You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.

2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: ERROR: No matching packages

2022-12-28 16:38:01,644 [ERROR] mlem.contrib.docker.utils: {'errorDetail': {'code': 1, 'message': "The command '/bin/sh -c pip install mlem-0.4.1.dev2+gd4e0ddb-py3-none-any.whl && pip cache purge' returned a non-zero code: 1"}, 'error': "The command '/bin/sh -c pip install mlem-0.4.1.dev2+gd4e0ddb-py3-none-any.whl && pip cache purge' returned a non-zero code: 1"}
Traceback (most recent call last):
  File "/Users/aguschin/Git/iterative/mlem-fly/mlem/contrib/docker/utils.py", line 146, in inner
    return f(*args, **kwargs)
  File "/Users/aguschin/Git/iterative/mlem-fly/mlem/contrib/docker/base.py", line 584, in build_image
    image, _ = build_image_with_logs(
  File "/Users/aguschin/Git/iterative/mlem-fly/mlem/contrib/docker/utils.py", line 44, in build_image_with_logs
    raise BuildError(chunk["error"], result_stream)
docker.errors.BuildError: The command '/bin/sh -c pip install mlem-0.4.1.dev2+gd4e0ddb-py3-none-any.whl && pip cache purge' returned a non-zero code: 1

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/aguschin/Git/iterative/mlem-fly/.venv/bin/mlem", line 8, in <module>
    sys.exit(app())
  File "/Users/aguschin/Git/iterative/mlem-fly/.venv/lib/python3.9/site-packages/typer/main.py", line 328, in __call__
    raise e
  File "/Users/aguschin/Git/iterative/mlem-fly/.venv/lib/python3.9/site-packages/typer/main.py", line 311, in __call__
    return get_command(self)(*args, **kwargs)
  File "/Users/aguschin/Git/iterative/mlem-fly/.venv/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/Users/aguschin/Git/iterative/mlem-fly/.venv/lib/python3.9/site-packages/typer/core.py", line 778, in main
    return _main(
  File "/Users/aguschin/Git/iterative/mlem-fly/.venv/lib/python3.9/site-packages/typer/core.py", line 216, in _main
    rv = self.invoke(ctx)
  File "/Users/aguschin/Git/iterative/mlem-fly/.venv/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/aguschin/Git/iterative/mlem-fly/.venv/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/aguschin/Git/iterative/mlem-fly/mlem/cli/main.py", line 174, in invoke
    return super().invoke(ctx)
  File "/Users/aguschin/Git/iterative/mlem-fly/.venv/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/aguschin/Git/iterative/mlem-fly/.venv/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/Users/aguschin/Git/iterative/mlem-fly/.venv/lib/python3.9/site-packages/typer/main.py", line 683, in wrapper
    return callback(**use_params)  # type: ignore
  File "/Users/aguschin/Git/iterative/mlem-fly/mlem/cli/main.py", line 475, in inner
    f(*iargs, **ikwargs)
  File "/Users/aguschin/Git/iterative/mlem-fly/mlem/cli/build.py", line 88, in build_type
    build(
  File "/Users/aguschin/Git/iterative/mlem-fly/mlem/telemetry.py", line 50, in inner
    return f(*args, **kwargs)
  File "/Users/aguschin/Git/iterative/mlem-fly/mlem/api/commands.py", line 333, in build
    return builder.build(model)
  File "/Users/aguschin/Git/iterative/mlem-fly/mlem/contrib/docker/base.py", line 564, in build
    return self.build_image(tempdir)
  File "/Users/aguschin/Git/iterative/mlem-fly/mlem/contrib/docker/utils.py", line 148, in inner
    raise MlemError(f"Error calling docker: {e}") from e
mlem.core.errors.MlemError: Error calling docker: The command '/bin/sh -c pip install mlem-0.4.1.dev2+gd4e0ddb-py3-none-any.whl && pip cache purge' returned a non-zero code: 1
```

--- a/mlem/contrib/docker/context.py
+++ b/mlem/contrib/docker/context.py
@@ -124,7 +124,7 @@ class WhlSource(MlemSource):
             f"COPY {MLEM_REQUIREMENTS} .\n"
             f"RUN pip install -r {MLEM_REQUIREMENTS} && pip cache purge\n"
             f"COPY {name} .\n"
-            f"RUN pip install {name} && pip cache purge || true\n"
+            f"RUN pip install {name} && (pip cache purge || true)\n"
         )
 
 

--- a/mlem/contrib/docker/context.py
+++ b/mlem/contrib/docker/context.py
@@ -124,7 +124,7 @@ class WhlSource(MlemSource):
             f"COPY {MLEM_REQUIREMENTS} .\n"
             f"RUN pip install -r {MLEM_REQUIREMENTS} && pip cache purge\n"
             f"COPY {name} .\n"
-            f"RUN pip install {name} && pip cache purge\n"
+            f"RUN pip install {name} && pip cache purge || true\n"
         )
 
 

--- a/tests/contrib/test_docker/test_context.py
+++ b/tests/contrib/test_docker/test_context.py
@@ -26,7 +26,7 @@ RUN pip install -r requirements.txt && pip cache purge
 COPY mlem_requirements.txt .
 RUN pip install -r mlem_requirements.txt && pip cache purge
 COPY {MLEM_LOCAL_WHL} .
-RUN pip install {MLEM_LOCAL_WHL} && pip cache purge
+RUN pip install {MLEM_LOCAL_WHL} && pip cache purge || true
 COPY . ./
 CMD sh run.sh
 """

--- a/tests/contrib/test_docker/test_context.py
+++ b/tests/contrib/test_docker/test_context.py
@@ -26,7 +26,7 @@ RUN pip install -r requirements.txt && pip cache purge
 COPY mlem_requirements.txt .
 RUN pip install -r mlem_requirements.txt && pip cache purge
 COPY {MLEM_LOCAL_WHL} .
-RUN pip install {MLEM_LOCAL_WHL} && pip cache purge || true
+RUN pip install {MLEM_LOCAL_WHL} && (pip cache purge || true)
 COPY . ./
 CMD sh run.sh
 """

